app/routes/goal_routes.py

```diff
@@ -1 +1,75 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, Response
+from app.models.goal import Goal
+from app.models.task import Task
+from ..db import db
+from .helper_methods import validate_model, helper_model_from_dict, helper_get_sorted_query
+
+bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
+
+
+
+@bp.post("")
+def create_goal():
+    request_body = request.get_json()
+    return helper_model_from_dict(Goal, request_body, required_fields=["title"])
+
+@bp.post("/<goal_id>/tasks")
+def add_tasks_to_goal_id(goal_id):
+    goal = validate_model(Goal, goal_id)
+    request_body = request.get_json()
+    
+    task_ids = request_body.get("task_ids", [])
+    # goal.tasks.clear()
+    for task in goal.tasks:
+        task.goal_id = None
+    
+    for id in task_ids:
+        task = validate_model(Task, id)
+        task.goal_id = goal.id
+        # goal.tasks.append(task)
```

Comment:
Take care to remove unnecessary commented out code before opening a PR for feedback, especially when you work in industry. 

Leaving behind code like this can lead to bugs if it gets uncommented and it raises confusion ("why is this commented out? why was this left in the code?"). If these comments are left in it can also make the codebase bloated because it would be filled with unnecessary lines of code. 

```suggestion
    for task in goal.tasks:
        task.goal_id = None
    
    for id in task_ids:
        task = validate_model(Task, id)
        task.goal_id = goal.id
```